<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteOrphan</class>
 <widget class="QWidget" name="WriteOrphan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>830</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="txtOrphan">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>761</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listPics">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>1251</width>
     <height>261</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
   <property name="batchSize">
    <number>6</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtBio">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>471</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Letter Body:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Orphan Biography:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attached Pictures:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAttachImage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>800</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Attach Picture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
